tensorflow_datasets/core/api_utils.py

```diff
@@ -19,12 +19,22 @@
 from __future__ import division
 from __future__ import print_function
 
+from typing import Any
+from typing import Callable
+from typing import Iterable
+from typing import List
+from typing import overload
```

Comment:
For functions, it's best not to use direct import:

```python
import typing
from typing import Any, Callable,...

@typing.overload
```

Reply:
Done